Automattic/wp-calypso
# Themes: allow to install premium themes in Jetpack sites

![image](https://user-images.githubusercontent.com/96308/27643573-bc0c97ca-5c21-11e7-944c-bf6f71712f8f.png)



This PR introduces front end updates that will make premium themes visible for Jetpack sites.

When the site is not in the Professional plan:
- a banner will be displaying prompting user to upgrade
- the Try & customize feature will be hidden
- a theme regular Activate button or menu item will be converted to Upgrade to activate and will lead user to Plans view.

When the site is in the Professional plan:
- the Try & customize feature will be available
- the Activate button or menu item will initiate the theme installation and will activate it

Testing instructions:

1. build this PR and browse themes. You must be able to see Premium themes on the Jetpack site. If the Jetpack site doesn't have a Jetpack Professional plan yet, you'll see labels "Upgrade to activate" instead of prices shown for wpcom sites. 
2. add a Jetpack Professional plan to your Jetpack site
3. to test theme installation, use Mood theme, is the only one that's built for now
4. Make sure any other showcase modes (logged-out, multi-site, wpcom-single-site, theme sheet) work as before.
5. Set `jetpack/pijp` to `false` in `development.json`, and restart Calypso. Verify that in Jetpack-single-site mode, the new functionality is removed, i.e. everything works as in `master`/`production`.
